pkg/operator/status.go

```diff
@@ -103,7 +103,12 @@ func (optr *Operator) syncStatus(conds []osconfigv1.ClusterOperatorStatusConditi
 			Name:      optr.name,
 		},
 		Status: osconfigv1.ClusterOperatorStatus{
-			Version: version.Raw,
```

Comment:
This belongs to a different commit. This effort it's being tracked here https://jira.coreos.com/browse/CLOUD-332

Reply:
I need to update it here because of `dep` updates to the latest changes, at least from CI I can see that it does not break anything.